src/simplify_dyn_ops.cpp

```diff
@@ -297,7 +326,8 @@ void simplify_dyn_ops::apply(module& m) const
                         find_static_2in_broadcasts{},
                         find_const_2in_slice{},
                         find_const_3in_slice{},
-                        find_const_4in_slice{});
+                        find_const_4in_slice{},
```

Comment:
Should all these find** operations happen in the sequence listed here..? Or can they be any amongst these..? Thanks.

Reply:
> Should all these find** operations happen in the sequence listed here..?

What do you mean by "happen"? The `find_matches` will pick the first one to match an instruction(skipping the rest of the matchers for that instruction). And, its still possible that all of these match in a `module` among other instructions. However, none of these matchers overlap in `simplify_dyn_ops` so order doesn't matter here, but it can matter if you have overlapping matchers.

Reply:
Thanks.